source/solvers/vof.cc

```diff
@@ -779,17 +757,13 @@ VolumeOfFluid<dim>::calculate_mass_deviation(
   const double                     sharpening_threshold)
 {
   // Copy present solution VOF
-  auto mpi_communicator = this->triangulation->get_communicator();
-
-  TrilinosWrappers::MPI::Vector solution_copy;
-  solution_copy.reinit(this->locally_owned_dofs, mpi_communicator);
-  solution_copy = this->present_solution;
+  evaluation_point = this->present_solution;
```

Comment:
so this does a copy also? or points to the same location?

Reply:
It's the equal operator, so it copies the content on the existing allocated memory space for evaluation point.